spec.html

```diff
@@ -2406,14 +2406,14 @@ <h1>The Object Type</h1>
       <p>An Object is logically a collection of properties. Each property is either a data property, or an accessor property:</p>
       <ul>
         <li>
-          A <dfn>data property</dfn> associates a key value with an ECMAScript language value and a set of Boolean attributes.
+          A <dfn variants="data properties">data property</dfn> associates a key value with an ECMAScript language value and a set of Boolean attributes.
         </li>
         <li>
-          An <dfn>accessor property</dfn> associates a key value with one or two accessor functions, and a set of Boolean attributes. The accessor functions are used to store or retrieve an ECMAScript language value that is associated with the property.
+          An <dfn variants="accessor properties">accessor property</dfn> associates a key value with one or two accessor functions, and a set of Boolean attributes. The accessor functions are used to store or retrieve an ECMAScript language value that is associated with the property.
         </li>
       </ul>
       <p>Properties are identified using key values. A property key value is either an ECMAScript String value or a Symbol value. All String and Symbol values, including the empty String, are valid as property keys. A <dfn id="property-name">property name</dfn> is a property key that is a String value.</p>
-      <p>An <dfn id="integer-index">integer index</dfn> is a String-valued property key that is a canonical numeric String (see <emu-xref href="#sec-canonicalnumericindexstring"></emu-xref>) and whose numeric value is either *+0*<sub>𝔽</sub> or a positive integral Number &le; 𝔽(2<sup>53</sup> - 1). An <dfn id="array-index">array index</dfn> is an integer index whose numeric value _i_ is in the range <emu-eqn>*+0*<sub>𝔽</sub> &le; _i_ &lt; 𝔽(2<sup>32</sup> - 1)</emu-eqn>.</p>
+      <p>An <dfn id="integer-index" variants="integer indexes">integer index</dfn> is a String-valued property key that is a canonical numeric String (see <emu-xref href="#sec-canonicalnumericindexstring"></emu-xref>) and whose numeric value is either *+0*<sub>𝔽</sub> or a positive integral Number &le; 𝔽(2<sup>53</sup> - 1). An <dfn id="array-index" variants="array indexes">array index</dfn> is an integer index whose numeric value _i_ is in the range <emu-eqn>*+0*<sub>𝔽</sub> &le; _i_ &lt; 𝔽(2<sup>32</sup> - 1)</emu-eqn>.</p>
```

Comment:
We should settle on one of "indexes" or "indices". There's 2 occurrences of "indexes" (1 of each of these plurals) currently and 11 of "indices".

Reply:
[ECMA-402](https://tc39.es/ecma402/) also has a single occurrence of "indices" and none of "indexes", and the [Segmenter proposal](https://tc39.es/proposal-intl-segmenter/) has two of "indices" and none of "indexes".

Reply:
I'm happy to change to be consistently "indices" everywhere.

Reply:
Let's just address it here. I don't think it's worth another issue/PR.